addon/styles/partials/_uni-select.scss

```diff
@@ -1,3 +1,29 @@
+.uni-select-wrapper--use-alias {
+  position: relative;
+
+  select.uni-select {
+    z-index: 2;
+    opacity: 0;
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+
+    &:focus + div.uni-select {
```

Comment:
is this `div.uni-select` really needed, asking 'cause it's an over-qualification of a selector. I know there's a `select.uni-select` but I believe we can improve on this and have a different class for each.